'''
So we use the GWAS SNP-IDs to get the Ensembl API data (the 1000G proj)
Since we are gonna immigrate them into 1 database we dont need the API we can just run a select statement and get all the SNP vals
Calc FST for all SNP populations


So I am just gonna go thru the tutorial and remake the funcs in py3 and then deal with integrating the backend
Everything is indeed in the graphing.r file
Input is a list of the genes
'''

def getSuperPops()
